SwiftWisdom/Core/UIKit/UIApplication+Extensions.swift

```diff
@@ -13,10 +13,14 @@ enum OpenSettingsError: Error {
 }
 
 extension UIApplication {
+
+    @available(iOS, unavailable, message: "Call UIApplication.open(_:) directly")
```

Comment:
If this function will now be a no-op when called, should we just make it `@available(*, ...)` since other platforms will not be able to gain anything from calling this?